libs/books/feature/src/lib/total-count/total-count.component.html

```diff
@@ -1,3 +1,3 @@
 <ng-container *ngIf="totalUnread$ | async as count">
-  <span matBadge="{{ count }}" matBadgeOverlap="false"> </span>
+  <span data-testing="total-book-count" matBadge="{{ count }}" matBadgeOverlap="false"> </span>
```

Comment:
It's not used anywhere. Please revert this change.

Reply:
Removed.